src/main/java/com/epam/training/restapp/controller/PostController.java

```diff
@@ -0,0 +1,46 @@
+package com.epam.training.restapp.controller;
+
+import com.epam.training.restapp.dto.PostInfo;
+import com.epam.training.restapp.service.PostService;
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RestController
+@AllArgsConstructor(onConstructor = @__(@Autowired))
+public class PostController {
+
+    private final PostService service;
+
+    @GetMapping("/posts")
```

Comment:
Move common mapping to Contoller level

Reply:
I have moved common mapping to a class level using @RequestMapping annotation